.env.app

```diff
@@ -1,6 +1,6 @@
 # feature flags
 REACT_APP_FEATURE_CHATWOOT=true
-REACT_APP_FEATURE_LIFI_SWAP=false
+REACT_APP_FEATURE_LIFI_SWAP=true
```

Comment:
```suggestion
```

Reply:
already set in base env

Reply:
Might as well keep it ? This was a revert of https://github.com/shapeshift/web/pull/6524 which disabled it on prod envs only.